protocol/messenger.go

```diff
@@ -6463,3 +6465,27 @@ func (m *Messenger) handleSyncSocialLinks(message *protobuf.SyncSocialLinks, cal
 func (m *Messenger) GetDeleteForMeMessages() ([]*protobuf.DeleteForMeMessage, error) {
 	return m.persistence.GetDeleteForMeMessages()
 }
+
+func (m *Messenger) ParseSharedURL(url string) (*MessengerResponse, error) {
+	if !strings.HasPrefix(url, baseStatusUrl) {
```

Comment:
Maybe convert `url` to lower case here? 
Or do we consider only everything in lower case as valid?